configure.ac

```diff
@@ -60,6 +60,7 @@ AC_PROG_LIBTOOL
 AC_PROG_RANLIB
 AC_HEADER_STDC
 AC_CHECK_HEADERS([infiniband/verbs.h],,[AC_MSG_ERROR([ibverbs header files not found])])
+AC_CHECK_HEADERS([sys/random.h],,[AC_MSG_ERROR([getrandom not found])])
```

Comment:
Sorry for this, I just checked witha  setup that doesn't have sys/random.h and it failed to compile,I think it should be checked with ac_try_link because now if we don't have sys/random.h the configuration will fail and the HAVE_SYS_RANDOM_H will not help us in this situation

Reply:
great, it solved the issue.
Thanks for your contribution.